version/description

```diff
@@ -0,0 +1,11 @@
+This release has rewrite the project from scratch using operator-sdk
```

Comment:
I want to see documentation of "how to do a release" somewhere in the project.

Reply:
Done